keps/74-support-argo-workflow/README.md

```diff
@@ -0,0 +1,807 @@
+# KEP-74: Support Generic Workflows
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [User Stories](#user-stories)
+    - [Story 1](#story-1)
+    - [Story 2](#story-2)
+    - [Story 3](#story-3)
+    - [Story 4](#story-4)
+- [Design Details](#design-details)
+  - [What are the Stages for Argo Workflows?](#what-are-the-stages-for-argo-workflows)
+  - [Workflow as An Unit](#workflow-as-an-unit)
+    - [How to suspend the whole workflow](#how-to-suspend-the-whole-workflow)
+    - [Drawback and Limitations](#drawback-and-limitations)
+    - [Advantages](#advantages)
+  - [Stage as An Unit](#stage-as-an-unit)
+    - [Workflow Framework in Kueue](#workflow-framework-in-kueue)
+    - [How to suspend by stages](#how-to-suspend-by-stages)
+    - [Advantages](#advantages-1)
+    - [Drawback and Limitations](#drawback-and-limitations-1)
+  - [Task as An Unit](#task-as-an-unit)
+    - [Advantages](#advantages-2)
+    - [Drawback and Limitations](#drawback-and-limitations-2)
+- [Additional Details](#additional-details)
+  - [Test Plan](#test-plan)
+    - [Unit Tests](#unit-tests)
+    - [Integration tests](#integration-tests)
+  - [Graduation Criteria](#graduation-criteria)
+<!-- /toc -->
+
+## Summary
+
+This KEP outlines the proposal to integrate Generic Workflows within Kueue, discussing the advantages 
+and disadvantages of queuing workflows at varying granularity levels, alongside detailing 
+the integration methodologies. Workflow managers like Argo Workflow and Tekton can follow the convention 
```

Comment:
```suggestion
the integration methodologies. Workflow managers like Argo Workflows and Tekton can follow the convention 
```

Reply:
Can you fix this in other places too?

Reply:
Thanks, I will explain the use cases. WIP.